avm/utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -59,7 +59,7 @@ Describe 'File/folder tests' -Tag 'Modules' {
 
     $moduleFolderTestCases = [System.Collections.ArrayList] @()
     foreach ($moduleFolderPath in $moduleFolderPaths) {
-      $resourceTypeIdentifier = ($moduleFolderPath -split '[\/|\\]{1}avm[\/|\\]{1}(res|ptn)[\/|\\]{1}')[2] -replace '\\', '/' # avm/res/<provider>/<resourceType>
+      $resourceTypeIdentifier = ($moduleFolderPath -split '[\/|\\]{1}avm[\/|\\]{1}(res|ptn)[\/|\\]{1}')[2] -replace '\\', '/' # <provider>/<resourceType>
```

Comment:
```suggestion
      $resourceTypeIdentifier = ($moduleFolderPath -split '[\/|\\]{1}avm[\/|\\]{1}(res|ptn)[\/|\\]{1}')[2] -replace '\\', '/' # avm/res|ptn/<provider>/<resourceType>
```

Don't think we should just remove the example 😄

Reply:
That's not the case. The value of `$resourceTypeIdentifier` is `<provider>/<resourceType>` and not `avm/res|ptn/<provider>/<resourceType>` due to the `split[2]`

Reply:
No no - it's an example what it matches against, not what the outcome is. When I'm seeing 
`[\/|\\]{1}avm[\/|\\]{1}(res|ptn)[\/|\\]{1}'`

I may we wondering what and actually string would look that the regex applies to. When I then get
`avm/res|ptn/<provider>/<resourceType>`

It helps me piece together: Ah ok a slash in the beginning, then AVM, then it looks for res or ptn, and finally another slash. With the string above that would mean if it splits by this value, what I get is ... `<provider>/<resourceType>`

Reply:
At least that's how I'm thinking. Which is why I added the examples in the first place :D

Reply:
If I see a comment next to an assignment I take for granted it is representing the value of the assignment, hence the confusion. Instead the comment refers to the regex and how it maps to the moduleFolderPath value, correct?

Reply:
Hey @eriqua , it shows how a 'moduleFolderPath' would look like. We could also change it to `# 'avm/res|ptn/<provider>/<resourceType>' would return '<provider>/<resourceType>'`

Reply:
Done @AlexanderSehr